src/components/ContentReveal/ContentReveal.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+
+import "./_content-reveal.scss";
+
+type Props = {
+  title: string | JSX.Element;
```

Comment:
I think this can be `ReactNode` to cover anything that react will render. Same below.